tests/filecheck/backend/x86/convert_big_param_func_to_x86.mlir

```diff
@@ -0,0 +1,6 @@
+// XFAIL: *
```

Comment:
I've never seen this before! We usually test for the diagnostic error message, can you please change it to that? the `--test-diagnostics` lets us not fail when a `DiagnosticException` is raised

Reply:
`--verify-diagnostics`